src/analysis/retail/shaman/enhancement/modules/normalizers/ProcNormalizer.tsx

```diff
@@ -2,22 +2,28 @@ import EventsNormalizer from 'parser/core/EventsNormalizer';
 import { AnyEvent, EventType } from 'parser/core/Events';
 import { Options } from 'parser/core/Analyzer';
 import { TALENTS_SHAMAN } from 'common/TALENTS';
+import SPELLS from 'common/SPELLS';
 
-/** This normalizer adds a fabricated cast event when deeply rooted elements procs
- * to simulate a cast of Ascendance for the major cooldown analyzer */
+/** This normalizer adds a fabricated cast event when deeply rooted elements or hot hand procs
+ * to simulate a cast for the major cooldown analyzers */
 
-const ASCENDANCE_ID = TALENTS_SHAMAN.ASCENDANCE_ENHANCEMENT_TALENT.id;
+const PROC_IDS = [TALENTS_SHAMAN.ASCENDANCE_ENHANCEMENT_TALENT.id, SPELLS.HOT_HAND_BUFF.id];
```

Comment:
fabricating / converting hot hands buffs into casts is likely to do very strange things and/or produce hard-to-debug bugs. what's the end-goal of converting hot hands into a cast? with ascendance there was the use case of integrating into the cooldown analyzers (which maybe should've been patched on the cooldown analyzer side).

Reply:
The idea was essentially the same so the MajorCooldown code could be resused.

I'm happy to look into modifying the MajorCooldown analyzer to support any ability instead.

Reply:
I'll create a separate PR to properly resolve this, along with the "hack" for Ascendance